### PR TITLE
chore: change line width to 80 for src/background

### PR DESCRIPTION
**Pull request by Shobhit1**
#### Description of changes
Run format check on `src/background` and update line width to 80. Also updated prettier config to include that folder in the override section.

#### Pull request checklist

- [x] Addresses an existing issue: #1713 
- [x] Ran `yarn fastpass`
- [x] Added/updated relevant unit test(s) (and ran `yarn test`)
- [ ] Verified code coverage for the changes made. Check coverage report at: `<rootDir>/test-results/unit/coverage`
- [ ] PR title *AND* final merge commit title both start with a semantic tag (`fix:`, `chore:`, `feat(feature-name):`, `refactor:`). Check workflow guide at: `<rootDir>/docs/workflow.md`
- [ ] (UI changes only) Added screenshots/GIFs to description above
- [ ] (UI changes only) Verified usability with NVDA/JAWS

### CODE REVIEW

File: src/background/actions/action-creator.ts [hunk 1]
```diff
@@ -55,21 +55,37 @@ export class ActionCreator {
...
+            actionHub.visualizationScanResultActions;
         this.inspectActions = actionHub.inspectActions;
     }
 
     public registerCallbacks(): void {
-        this.interpreter.registerTypeToPayloadCallback(visualizationMessages.Common.Toggle, this.onVisualizationToggle);
-        this.interpreter.registerTypeToPayloadCallback(visualizationMessages.Common.ScanCompleted, this.onScanCompleted);
-        this.interpreter.registerTypeToPayloadCallback(visualizationMessages.Common.ScrollRequested, this.onScrollRequested);
-        this.interpreter.registerTypeToPayloadCallback(visualizationMessages.Common.RescanVisualization, this.onRescanVisualization);
+        this.interpreter.registerTypeToPayloadCallback(
```
Comment (smoralesd) [inline src/background/actions/action-creator.ts]: unneeded extra new lines

Comment (Shobhit1) [reply src/background/actions/action-creator.ts]: This is just how `GitHub` is showing this. There are no extra lines here. You can also check the line numbers here or the attached screenshot from my local code instance.

image

File: src/background/stores/card-selection-store.ts [hunk 2]
```diff
@@ -71,14 +94,18 @@ export class CardSelectionStore extends BaseStoreImpl<CardSelectionStoreData> {
...
+                payload.resultInstanceUid
+            ] === undefined
         ) {
             return;
         }
 
         const rule = this.state.rules[payload.ruleId];
 
-        rule.cards[payload.resultInstanceUid] = !rule.cards[payload.resultInstanceUid];
+        rule.cards[payload.resultInstanceUid] = !rule.cards[
```
Comment (DaveTryon) [inline src/background/stores/card-selection-store.ts]: I get the reasons for the shorter line lengths, but it feels like we have some cases where we sacrifice the ease of reading the code. This line is just one example--the old code is obvious that we're flipping the state of `rule.cards[payload.resultInstanceUid]`. In the new code I need to "re-stitch" across 3 lines.

File: src/background/stores/global/scoping-store.ts [hunk 3]
```diff
@@ -41,7 +44,12 @@ export class ScopingStore extends BaseStoreImpl<ScopingStoreData> {
     private onAddSelector = (payload: ScopingPayload): void => {
         let shouldUpdate: boolean = true;
         _.forEach(Object.keys(this.state.selectors[payload.inputType]), key => {
-            if (_.isEqual(this.state.selectors[payload.inputType][key], payload.selector)) {
+            if (
```
Comment (DaveTryon) [inline src/background/stores/global/scoping-store.ts]: This strikes me as another case where the "vertical expansion" isn't particularly helpful

File: src/background/stores/card-selection-store.ts (hunk 2 above)
Comment (Shobhit1) [reply src/background/stores/card-selection-store.ts]: All of these are part of automated prettier `format:fix` run. So, I didn't control this except for changing line width to 80 in the prettier config. But that being said, I feel we should discuss this but I am not unsure if we want to block this PR or not.

**Comment by Shobhit1** [comment]
Closing this in favor of choosing between 100 & 120 as our line width.

**Pull request closed without merging**